Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationEntry.cs

```diff
@@ -101,6 +102,31 @@ public static ContentLocationEntry TryCreateFromRedisValue(byte[] data, UnixTime
             return FromRedisValue(data, lastAccessTime);
         }
 
+        /// <summary>
+        /// Serializes an instance into a binary stream.
+        /// </summary>
+        public void Serialize(BuildXLWriter writer)

```

Comment:
BTW, I think it make sense to move to BuildXLWriter everywhere, because it has some extra features required for high perf implementaiton.